tests/common.go

```diff
@@ -2971,19 +2976,21 @@ func CreateS3BackupLocation(name string, uid, cloudCred string, cloudCredUID str
 	}
 	//ctx, err := backup.GetPxCentralAdminCtx()
 	ctx, err := backup.GetAdminCtxFromSecret()
-	expect(err).NotTo(haveOccurred(),
-		fmt.Sprintf("Failed to fetch px-central-admin ctx: [%v]",
-			err))
+	if err != nil {
+		return err
+	}
 	_, err = backupDriver.CreateBackupLocation(ctx, bLocationCreateReq)
-	if err != nil && strings.Contains(err.Error(), "already exists") {
-		return
+	if err != nil {
+		return err
 	}
-	expect(err).NotTo(haveOccurred(),
-		fmt.Sprintf("Failed to create backuplocation [%s] in org [%s]", name, orgID))
+	if err != nil {
+		return fmt.Errorf("failed to create backup location: %v", err)
+	}
```

Comment:
There are 2 `if err != nil` here, just keep the one that returns more verbose error

Reply:
Added by mistake. Now corrected it